lib/forty_facets/filter/facet_filter_definition.rb

```diff
@@ -49,7 +49,7 @@ class AttributeFilter < FacetFilter
       def selected
         entity = definition.origin_class
         column = entity.columns_hash[definition.attribute.to_s]
-        values.map{|v| column.type_cast(v)}
+        values.map{|v| entity.connection.type_cast(v, column)}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.